modules/dists/ReplicatedDist.chpl

```diff
@@ -424,6 +424,16 @@ class ReplicatedArr : BaseArr {
         halt("locale ", here.id, " has no local replicand");
     return localArrs[here.id];
   }
+
+  /* Access another locale's local array representation by locale id */
+  proc replicand(id: here.id.type) ref {
```

Comment:
I'll add that I don't know yet that I agree with Michael's philosophy.  For example `student.height(unit=inches)` seems like a reasonable call to support to me since it's querying a property of a student (something we might consider field-like).  In contrast, `student.getHeight(unit=inches)` feels a little unnecessarily verbose.  OTOH, I suppose it could be argued that `student.height(unit=inches)` leaves it unclear whether it's a setter or a getter (though in Chapel, it could be both...).

Anyway, I'm still wrestling with this and just noting the thought, not insisting on the renaming.